config/config.go

```diff
@@ -740,6 +768,9 @@ type ClusterUser struct {
 	// if omitted or zero - no limits would be applied
 	AllowedNetworks Networks `yaml:"-"`
 
+	// A wildcarded user is mapped to this cluster user
+	// IsWildcarded bool
```

Comment:
Why is it commented?

Reply:
It is commented out because we do not need this flag for cluster user at config level, it is in clusterUser struct in scope.go
Removed. 
Thank you for noticing.